### PR TITLE
14.0 mig l10n fi edicode

### PR DESCRIPTION
**Pull request by jarmokortetjarvi**
No functional changes between 13.0 and 14.0.
Added simple tests for partner and edicode operator.
Changed `Liaison Technologies Oy` to `Opentext Oy`

Changes between `13.0` and `14.0`
https://github.com/jarmokortetjarvi/l10n-finland/commit/2093ab90a24fce85b8c508eba2ee1d734fd9815e
and
https://github.com/OCA/l10n-finland/compare/13.0...jarmokortetjarvi:14.0-mig-l10n_fi_edicode

### CODE REVIEW

**Comment by aisopuro** [comment]
The changes LGTM. 

Looking at the coverage comments, it looks like adding tests for `name_get` (I.e. looking at `display_name`) and doing a `name_search` should get the necessary coverage.

File: l10n_fi_edicode/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see http://www.gnu.org/licenses/agpl.html
+#
+{
+    "name": "Partner EDI code",
+    "summary": "Adds EDI code field and operators",
+    "version": "14.0.1.0.0",
```
Comment (mlaitinen) [inline l10n_fi_edicode/__manifest__.py:22]: Version here doesn't match the version in `readme/HISTORY.rst`

**Review by mlaitinen — CHANGES_REQUESTED** [review]

**Review by mlaitinen — APPROVED** [review]

**Comment by jarmokortetjarvi** [comment]
Ping @aisopuro for a re-approve

**Review by aisopuro — APPROVED** [review]
SLGTM.

**Comment by aisopuro** [comment]
> Sorry @aisopuro you are not allowed to merge.
> 
> To do so you must either have push permissions on the repository, or be a declared maintainer of all modified addons.
> 
> If you wish to adopt an addon and become it's [maintainer](https://odoo-community.org/page/maintainer-role), open a pull request to add your GitHub login to the `maintainers` key of its manifest.

Brutal. I guess it's all you @mlaitinen

**Pull request merged**